include/ofxPropsBySet.h

```diff
@@ -0,0 +1,733 @@
+// Copyright OpenFX and contributors to the OpenFX project.
+// SPDX-License-Identifier: BSD-3-Clause
+// NOTE: This file is auto-generated by gen-props.py. DO NOT EDIT.
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <map>
+#include <array>
+#include "ofxImageEffect.h"
+#include "ofxGPURender.h"
+#include "ofxColour.h"
+#include "ofxDrawSuite.h"
+#include "ofxParametricParam.h"
+#include "ofxKeySyms.h"
+// #include "ofxOld.h"
+
+namespace OpenFX {
+// Properties for property sets
+const std::map<const char *, std::vector<const char *>> prop_sets {
```

Comment:
again, here you should consider consistency, pick `std::string` or `const char*`